docs/content/documentation/systems/AuxScalarKernels/tensor_mechanics/GeneralizedPlaneStrainReferenceResidual.md

```diff
@@ -0,0 +1,14 @@
+# GeneralizedPlaneStrainReferenceResidual
+!syntax description /AuxScalarKernels/GeneralizedPlaneStrainReferenceResidual
+
+## Description
+The AuxScalarKernel `GeneralizedPlaneStrainReferenceResidual` retrieves a reference residual value from `GeneralizedPlaneStrainUserObject` for use with `ReferenceResidualProblem`.
```

Comment:
@jasondhales would you provide links to the classes referenced here please? That way a user can click on each of the referenced classes for more information.